plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -532,4 +533,17 @@ public DeltaLakeConfig setDeletionVectorsEnabled(boolean deletionVectorsEnabled)
         this.deletionVectorsEnabled = deletionVectorsEnabled;
         return this;
     }
+
+    public boolean isDeltaLogFileSystemCacheEnabled()
+    {
+        return deltaLogFileSystemCacheEnabled;
+    }
+
+    @Config("delta.fs.cache.delta-log-caching-enable")
+    @ConfigDescription("Enable object storage caching of the _delta_log directory (effective only when fs.cache.enabled=true)")
```

Comment:
I like "filesystem" more, but I used "object storage" because the description of `fs.cache.enabled` mentions "object storage", i.e. "Enable object storage caching. Defaults to no caching with the value false." Should the documentation use filesystem here as well?

Reply:
filesystem is the more accurate term as it's not limited to object stores and applies to HDFS as well

Reply:
I renamed it to "filesystem" here. I think I need a new PR to fix the description on the `fs.cache.enabled` property, am I right?